cvIgnacioCalahorro.html

```diff
@@ -89,30 +83,28 @@ <h2 class='about'>ABOUT ME:</h2>
                 <p class="ses">Studies</p>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc GIS at Esri Spain 
+                        MSc in GIS at Esri Spain 
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Graduated expected on May 2022
+                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Expected to graduate in May 2022
                     </p>
                 </div>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc in Surveying and Mapping.
+                        MSc in Surveying and Mapping at Aalborg University, DK
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Aalborg University, DK. Graduated 2020 
+                        &nbsp;&nbsp;&nbsp;&nbsp;Graduated 2020 
                     </p>
                     <p>
-                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture.
-                        At the first year. Where I was working on the first semester project analyzing a point cloud,
-                        vectorize and compare measurements made with total station with the ones calculate by a code
-                        developed in Python.
+                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture. 
+                        I took part in the programme in the first semester of the studies by working on a project which involved point cloud analysis and total station measurement vectorization and comparison. The whole project was carried out in the Python programming language.
                     </p>
                     <p>
-                        &nbsp;&nbsp;On the second semester I am working on a Indoor Positioning project I am working with the wiffi signal and try to find a cheap way to implement the precision. Also during this semester the main topic of the subject are Statistic and Data Fusion and Positioning Technologies.
+                        &nbsp;&nbsp;In the second semester, I worked on an Indoor Positioning project utilizing Wi-Fi signal to triangulate the position of the object in question TODO: do it cheaply. The main topic of the supportive courses for the project were Statistics and Data Fusion and Positioning Technologies.
```

Comment:
- "on" -> "in" the second semester
- "I am working" -> "I worked", it's past tense
- "a" -> "an", when the next word starts with a vowel (or a silent letter followed by a vowel), you go for "an"
- then the rest of the sentence is rewritten to sound sexier
- I just now noticed that I left a "TODO" there, sorry about that. You have info on why is there a TODO on Telegram